app/controllers/screenings_controller.rb

```diff
@@ -13,7 +13,13 @@ def show
   end
 
   def new
-    @screening = @movie.screenings.by_user(current_user).new
+    default_params = {
+      date_watched: Time.zone.today,
```

Comment:
Can use `Date.current`. Basically it'll use `Time.zone` if a timezone is defined: https://stackoverflow.com/questions/6635363/what-is-the-difference-between-date-current-and-date-today